scripts/approvalYears.py

```diff
@@ -222,15 +225,15 @@ def apprDateRegression(prods):
                 datays[kind2].append(ts)
 
     models = dict()
-    for kind in dataxs.keys():
+    for kind in list(dataxs.keys()):
         #print kind, len(dataxs[kind])
         if len(dataxs[kind]) > 20:
             regr = numpy.poly1d(numpy.polyfit(dataxs[kind], datays[kind], 1))
             models[kind] = regr
     figs = ['NDA', 'BLA', 'ANDA']
     for figt in figs:
         fig, ax = plt.subplots()
-        for kind in models.keys():
+        for kind in list(models.keys()):
```

Comment:
Same as above.

Reply:
Python3 versus python2 issue. 2to3 automagically has made this change as .keys() no longer returns a list in python3